dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -170,4 +170,7 @@ servletPathForDefaultMapping.name=Set the servlet path value for a default mappi
 servletPathForDefaultMapping.desc=Set the servlet path value to the request URI minus the context path. The path information is null when a servlet is used as a default mapping. The default value is true for version 4.0 or later of the servlet feature. It is false for other servlet features. When mapping is to the /* pattern, the servlet path is empty and the path information starts with a leading slash (/).
 
 throwExceptionWhenUnableToCompleteOrDispatch.name=Throw exception for not complete or dispatch asyn
-throwExceptionWhenUnableToCompleteOrDispatch.desc=Throw an illegal state exception when an asynchronous request cannot be completed or dispatched. The default is true. If the asynchronous request must complete or the dispatch method must return even if the call does not succeed, set the property to false.
\ No newline at end of file
+throwExceptionWhenUnableToCompleteOrDispatch.desc=Throw an illegal state exception when an asynchronous request cannot be completed or dispatched. The default is true. If the asynchronous request must complete or the dispatch method must return even if the call does not succeed, set the property to false.
+
+getRealPathReturnsQualifiedPath.name=Always return a non-null path from getRealPath
+getRealPathReturnsQualifiedPath.desc=Return non-null paths from ServletContext.getRealPath(String), even if no resource exists at the given path. The default is true. If applications expect getRealPath to return null when given a path for which no resource exists, set the property to false.
```

Comment:
**change**
Return non-null paths from ServletContext.getRealPath(String), even if no resource exists at the given path. The default is true. If applications expect getRealPath to return null when given a path for which no resource exists, set the property to false. 
**to**
Return non-null paths from the getRealPath method, even if no resource exists at the given path. The default is true. If applications expect the getRealPath method to return null when they are given a path for which no resource exists, set the property to false.